.bumpversion.cfg

```diff
@@ -34,46 +34,10 @@ message = Bump version: {current_version} → {new_version}
 search = version="{current_version}"
 replace = version="{new_version}"
 
-[bumpversion:file:checkbox-ng/checkbox_ng/__init__.py]
-search = __version__ = '{current_version}'
-replace = __version__ = '{new_version}'
-
-[bumpversion:file:checkbox-ng/plainbox/__init__.py]
-search = __version__ = '{current_version}'
-replace = __version__ = '{new_version}'
```

Comment:
If this is removed, then the current release process tools will leave `__version__` in `checkbox-ng/plainbox/__init__.py` as "2.6" forever...

Reply:
`checkbox-ng/plainbox/__init__.py` should get the version number at runtime now so this line should be safe to delete